x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/index.tsx

```diff
@@ -125,6 +132,14 @@ export const EndpointDetailsFlyout = memo(() => {
 
 EndpointDetailsFlyout.displayName = 'EndpointDetailsFlyout';
 
+const PolicyResponseFlyout = styled.div`
```

Comment:
😨 
Just want to confirm that this deviation from Eui is absolutely needed.

cc:/ @bfishel

Reply:
there is a lot of space in between the back to endpoint details link and the beginning of the policy response, which is why the padding adjusted was added